docs/pages/upgrading/cloud-kubernetes.mdx

```diff
@@ -4,127 +4,111 @@ description: Provides instructions for upgrading Teleport Cloud agents that run
 ---
 
 This guide explains how to upgrade Teleport Cloud agents running on Kubernetes.
-On Teleport Cloud, Auth Service and Proxy Service upgrades are managed for you.
-To keep agents up to date, you can either enroll them in automatic updates or
-upgrade them manually using the method you used to install Teleport.
+To keep agents up to date, you can either perform:
+- [Automatic updates](#automatic-updates) (recommended)
+- [Manual updates](#manual-updates)
 
 ## Prerequisites
 
 - Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
   which describes the sequence in which to upgrade components of your cluster.
 
-- Teleport Enterprise Cloud. You can determine the current version of these
-  services by running the following command, where `mytenant` is the name of
-  your Teleport Enterprise Cloud tenant. This requires the
-  [`jq` CLI tool](https://jqlang.github.io/jq/):
-
-  ```code
-  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.server_version'
-  ```
-
 - (!docs/pages/includes/cloud/tctl-tsh-prerequisite.mdx!)
 
 - (!docs/pages/includes/tctl.mdx!)
 
-Teleport supports automatic agent updates for the `teleport-kube-agent` Helm
-chart. The [Automatic Update Architecture
-](../architecture/agent-update-management.mdx) guide describes how agent
-updating works. Automatic agent upgrades require:
+## Check for agent updates
+
+The latest compatible Teleport agent version is served by the Teleport Proxy. To
+identify what that version is, you can query the `webapi` endpoint.
+```code
+$ curl https://<Var name="teleport.example.com" />/webapi/automaticupgrades/channel/stable/cloud/version
+v15.2.1
+```
+
+To list the agents that are upgradable, use the `tctl inventory ls` command to list
+the inventory of connected agents. The agents are listed along with their current
+version. Additional flags are available to filter by service `--services=node,kube,db,...`,
+or to filter by version `--older-than=v15.2.1`.
```

Comment:
> Additional flags are available to filter by service `--services=node,kube,db,...`,
or to filter by version `--older-than=v15.2.1`.

I would expand this sentence and use conventional punctuation and sentence structure. Right now, this looks fairly jumbled.